resources/css/style.css

```diff
@@ -1,196 +1,249 @@
 @import url("https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700;900&family=Poppins:wght@400;700;900&display=swap");
-
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@800&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap");
 
+/* Global rules */
 html {
   font-size: 62.5%;
 }
 
-html,
 body {
-  height: 100vh;
   box-sizing: border-box;
-  line-height: normal;
 }
 
-a {
-  text-decoration: none;
+html,
+body {
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+img {
+  max-width: 100%;
+  display: block;
 }
 
-.main-topic {
+h1 {
   font-family: "Playfair Display", serif;
-  font-weight: 400;
   font-size: 5rem;
+  line-height: 5rem;
+  font-weight: 400;
+  color: #1a2236;
+}
+
+@media (min-width: 768px) {
+  h1 {
+    font-size: 9rem;
+    line-height: 100%;
+  }
+}
+
+@media (min-width: 1024px) {
+  h1 {
+    font-size: 11rem;
+    line-height: 100%;
+  }
+}
+
+h3 {
+  font-family: Poppins, serif;
+  font-size: 2rem;
+  font-weight: 300;
+  line-height: 1.6rem;
+  color: #fff;
+}
+
+p {
+  font-family: Poppins, serif;
+  font-weight: 400;
+  font-size: 1.4rem;
+  line-height: 2.1rem;
+}
+
+a,
+button {
+  text-decoration: none;
+  font-family: Poppins, serif;
+  font-size: 1.6rem;
+  font-weight: 500;
+  color: #fff;
 }
 
-.top-nav-burger {
+/* 1. About me section" */
+
+.menu-top {
   display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding-right: 1.5rem;
 }
 
-.container.introduction {
-  background-image: url("/resources/images/image_geometry_2.png");
-  background-repeat: no-repeat;
-  background-position: bottom;
+.desktop-nav-top {
+  display: none;
 }
 
-.grd-container {
-  display: grid;
-  justify-content: center;
+@media (min-width: 768px) {
+  .menu-top {
+    display: none;
+  }
+
+  .desktop-nav-top {
+    display: block;
+  }
+
+  .desktop-nav-list {
+    display: flex;
+    justify-content: center;
+    gap: 5rem;
+    padding: 3.8rem 26.5rem;
+    border: 1px solid black;
+  }
 }
 
-.works-catalog {
-  grid-template-rows: repeat(9, auto auto);
+.background-image-nav {
+  background-image: url("./resources/images/image_geometry_1.png");
+  width: 100%;
+  height: 7.4rem;
+  background-repeat: no-repeat;
+  min-height: fit-content;
 }
 
-.introduction {
+.burguer-image {
+  padding: 1rem 1rem 1rem 0;
+}
+
+.about-me {
   display: flex;
-  max-width: 120rem;
-  margin: 0 auto;
   flex-direction: column;
-  min-height: 85vh;
-  align-items: center;
-  padding: 2.5rem;
+  align-content: space-between;
+  height: 90vh;
+  background-image: url("./resources/images/image_geometry_2.png");
```

Comment:
## OPTIONAL 

- [ ] Kindly use this format instead for all broken links `../images/image_geometry_2.png` currently the images are still not visible. please try this format above, If you have noticed images not visible was added as background images with CSS 👍🏿 

- Please do this for all images not currently visible in your project 👍🏿